web/server/codechecker_server/session_manager.py

```diff
@@ -195,6 +196,21 @@ def __init__(self, configuration_file, root_sha, force_auth=False):
         # Save the root SHA into the configuration (but only in memory!)
         self.__auth_config['method_root'] = root_sha
 
+        try:
+            self.__regex_groups_enabled = \
+                self.__auth_config['regex_groups'].get('enabled')
+        except KeyError:
+            self.__regex_groups_enabled = False
+
+        # Pre-compile the regular expressions of 'regex_groups'
+        if 'regex_groups' in self.__auth_config:
```

Comment:
Maybe we could simplify this code on the following way:
```suggestion
        self.__regex_groups_enabled = False

        # Pre-compile the regular expressions of 'regex_groups'
        if 'regex_groups' in self.__auth_config:
            self.__regex_groups_enabled = \
                self.__auth_config['regex_groups'].get('enabled', False)
```